### PR TITLE
Add check script for remote datasets

### PR DESCRIPTION
**Pull request by PaulWessel**
Make plots of both 05m and 06m resolutions for all datasets since that exercises both single grids and tiles.  All plots look good now.

### CODE REVIEW

File: scripts/remote_map_check.sh [hunk 1]
```diff
@@ -0,0 +1,20 @@
+#!/bin/bash -e
+# remote_map_check.sh - Make global test maps for both 05m and 06m resolutions
+#
+# usage: remote_map_check
+#
+# Loops over all recipies and makes global maps of them for the two resolutions
+# This checks that the tiled and single grid versions
```
Comment (meghanrjones) [inline scripts/remote_map_check.sh:7]:
```suggestion
# This checks that the tiled and single grid versions are similar
```

**Review by meghanrjones — APPROVED** [review]
Why is the default colorscale for earth_mask_05m and earth_mask_06m different?

**Comment by PaulWessel** [comment]
Not sure, looking at that.  There is no CPT listed for mask so I assume it gets turbo.

**Comment by PaulWessel** [comment]
So it is because the 5m grid has a node or two that catches level 4 (that pond in a lake in NA somewhere) whereas 6m has lost that detail and stops as level 3.  Also, because this data is not tiled (the biggest file for 15 arc sec mask is only 7.5M and most are less than 1 Mb (using byte and the compression detects large swaths of constants in the 0-4 range and compresses very well!), the test is a bit meaningless.

**Comment by meghanrjones** [comment]
Sounds good, thanks for the explanation.

**Pull request merged**